test/common/testlib.py

```diff
@@ -784,9 +788,6 @@ def skipImage(reason, *args):
     return lambda func: func
 
 class Policy(object):
-    def __init__(self):
-        self.retryable = True
```

Comment:
When `opts.thorough` is true, no Policy is used, and no retries will be done.  Is that enough for now?